src/olympia/addons/tests/test_views.py

```diff
@@ -3395,8 +3490,8 @@ def test_duplicate_version_number_error(self):
             self.url,
             data=self.minimal_data,
         )
-        assert response.status_code == 400, response.content
-        assert response.data == {
+        assert response.status_code == 409, response.content
+        assert response.json() == {
```

Comment:
out of interest, why the change to `response.json()` from `response.data`? `response.data` seems to be - by far- the most common use in our tests.

Reply:
`response.data` is the python representation of what _would_ be rendered, but can still contain objects like `ErrorDetail`. `response.json()` is the rendered json converted back to Python. The latter is easier to compare to deep dicts.